app/components/Dropdown/index.js

```diff
@@ -29,33 +36,50 @@ export const Dropdown = ({
   isArrowed,
   isMenuLabelMobile,
   isArrowMarginMobile,
-}) => (
-  <DropdownStyled
-    className={`dropdown show ${className}`}
-    isMenuLabelMobile={isMenuLabelMobile}
-    isArrowMarginMobile={isArrowMarginMobile}
-  >
-    <button
-      id={id}
-      className="d-flex align-items-center w-100 p-0"
-      type="button"
-      data-toggle="dropdown"
-      aria-haspopup="true"
-      aria-expanded="false"
-      data-icon="arrow"
-    >
-      {button}
+}) => {
+  const [isOpen, setIsOpen] = useState(false);
+  //const toggleHandler = () => setIsOpen(prevIsOpen => !prevIsOpen);
+  const toggleHandler = () => setIsOpen(!isOpen ? true : false);
 
-      {isArrowed && (
-        <ArrowDown icon={arrowDownIcon} width="10" alt="data-icon" />
+  return (
+    <DropdownStyled
+      onClick={toggleHandler}
+      className={`dropdown show ${className}`}
+      isMenuLabelMobile={isMenuLabelMobile}
+      isArrowMarginMobile={isArrowMarginMobile}
+    >
+      <button
+        id={id}
+        className="d-flex align-items-center w-100 p-0"
+        type="button"
+        //data-toggle="dropdown" //!!!! Открывает меню !!!!
+        aria-haspopup="true"
+        aria-expanded={'false'}
+        data-icon="arrow"
+      >
+        {button}
+        {isArrowed && (
+          <ArrowDown
+            onClick={toggleHandler}
+            open={isOpen}
+            icon={arrowDownIcon}
+            width="10"
+            alt="data-icon"
+          />
+        )}
+      </button>
+      {isOpen && (
+        <MenuStyled
```

Comment:
I don't think it is a preffered approach.
Styles of `MenuStyled` component can be changed. Or create a new component if needed.
@ruslanzharkov @Lampardks ^^